modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -8,6 +8,11 @@
 
 As a cluster administrator, you can disable IPsec encryption only if you enabled IPsec after cluster installation.
 
+[IMPORTANT]
+====
+To avoid issues with your installed cluster you disable IPsec, ensure that you delete the associated IPsec daemonsets pods.
```

Comment:
Good catch. Updated to 

> To avoid issues with your installed cluster, ensure that after you disable IPsec that you also delete the associated IPsec daemonsets pods.